runtime-sdk/src/modules/accounts/mod.rs

```diff
@@ -174,6 +174,20 @@ pub trait API {
     fn get_total_supplies<S: storage::Store>(
         state: S,
     ) -> Result<BTreeMap<token::Denomination, u128>, Error>;
+
+    // Move amount from address into fee accumulator.
+    fn move_into_fee_accumulator<C: Context>(
+        ctx: &mut C,
+        from: Address,
+        amount: &token::BaseUnits,
+    ) -> Result<(), modules::core::Error>;
+
+    // Move amount from fee accumulator into address.
+    fn move_from_fee_accumulator<C: Context>(
+        ctx: &mut C,
+        to: Address,
+        amount: &token::BaseUnits,
+    ) -> Result<(), modules::core::Error>;
```

Comment:
Exposing this externally can be quite dangerous. We should maybe consider just getting rid of this and transfer directly to the fee accumulator account (after measuring any potential performance impact, if it's not too big). Can you file an issue?

Reply:
Filed as https://github.com/oasisprotocol/oasis-sdk/issues/433 :)